docs/api-status.json

```diff
@@ -1531,6 +1531,10 @@
         "name": "Image.Resize",
         "comment": "Resize an rbd image.\n\nImplements:\n int rbd_resize(rbd_image_t image, uint64_t size);\n"
       },
+      {
```

Comment:
You seem to have added this manually to the wrong section. We usually add new APIs to the "preview" section first. Please see: https://github.com/ceph/go-ceph/blob/master/docs/development.md#api-status

Reply:
Indeed. I totally overlooked this somehow.

Reply:
Also, please avoid moving the existing function for now. Like @ansiwen mentioned we need to mark the content of the new file `preview` initially. Later when resize.go is no longer preview we can relocate the existing resize function.

Reply:
Accommodated!

Reply:
Thanks for the update. However, there are still resize2 callback related stuff in rbd.go. Can you please move those items to the new file as well? Thanks!

Reply:
Just fixed, sorry about that!